packages/react-components/package.json

```diff
@@ -534,4 +538,4 @@
     "./renderers/useSimpleRenderer.js": "./renderers/useSimpleRenderer.js",
     "./renderers/useSimpleRenderer.js.map": "./renderers/useSimpleRenderer.js.map"
   }
-}
\ No newline at end of file
+}
```

Comment:
Looks like `npm run build` always reverts this change. Maybe it's better to keep it as it was to avoid unintentional local changes.

Reply:
Done.